packages/app/src/pages/dashboard/_/menu/import/index.tsx

```diff
@@ -55,7 +58,7 @@ const Import: React.FC<Props> = ({ className = '' }) => {
           endpointGroupList={distribution.endpointGroupList || []}
         />
       </Modal>
-      <Error {...error.bind} />
+      <Error.modal {...error.bind} onClose={handleErrorModalOnClose} />
```

Comment:
This onClose implementation seems like it would basically always be the same, so it might be better to include it in the bind variable from the start.

However, there might be a problem with returning it under the name onClose, as useError cannot know whether it's used in a modal.

Due to this reason, useError and ErrorModal might be loosely coupled, but it's concerning that the same implementation needs to be repeated. Is there any good idea?